src/server/models/tag-campaign-contact.js

```diff
@@ -0,0 +1,28 @@
+import thinky from "./thinky";
+import { requiredString, timestamp } from "./custom-types";
+const type = thinky.type;
+import CampaignContact from "./campaign-contact";
+import Tag from "./tag";
+
+const TagCampaignContact = thinky.createModel(
+  "tag_campaign_contact",
+  type
+    .object()
+    .schema({
+      id: type.string(),
+      value: type.string(),
+      tag_id: requiredString(),
+      campaign_contact_id: requiredString(),
+      created_at: timestamp(),
+      updated_at: timestamp()
+    })
+    .allowExtra(false),
+  {
+    noAutoCreation: true,
+    depenencies: [Tag, CampaignContact]
+  }
+);
+
+TagCampaignContact.ensureIndex("campaign_contact_id");
```

Comment:
These don't do anything right now and knex migrations are the source of truth, so I would remove them 

cc/ @schuyler1d. If we want to continue adding these for some reason, then let's merge this PR: https://github.com/MoveOnOrg/rethink-knex-adapter/pull/28